### PR TITLE
Setonix custom amber

### PR DESCRIPTION
**Pull request by pelahi**
Updates the modules produced to add arpack-ng and then has script that defines bash functions to install amber.

### CODE REVIEW

File: systems/setonix/custom/amber_cpu.sh [hunk 1]
```diff
@@ -0,0 +1,289 @@
+
+# define the installation path 
+if [ ! -z ${AMBER_INSTALL_DIR} ]
+then 
+  export AMBER_INSTALL_DIR="/software/setonix/${DATE_TAG}/custom/software/zen3/gcc/12.1.0/amber/2022"
```
Comment (dipietrantonio) [inline systems/setonix/custom/amber_cpu.sh:5]: Change `gcc/12.1.0` to `gcc/12.2.0`

**Comment by dipietrantonio** [comment]
We should think about merging our two scripts somehow: https://gist.github.com/dipietrantonio/1f2755ae46c4b6119d2b1c540843c159

**Comment by ddeeptimahanti** [comment]
@pelahi , arpack-ng is a hidden module on Setonix

```
ddeeptimahanti@setonix-01:~> module --show_hidden av arpack

----------------------------- /software/setonix/2023.08/modules/zen3/gcc/12.2.0/dependencies -----------------------------
   .arpack-ng/3.8.0-fz75dzy (H)
```

You need to change systems/setonix/custom/amber_22_custom.sh script to load it correctly

**Comment by pelahi** [comment]
arpack-ng shouldn't be though (as is used by other packages and useful for custom builds). Hence the update to the yamls file. Does mean we'll need to manually tweak the config but that is fine and rebuild the module for arpack-ng. The PR is fine though since it has the update with the recipe.

**Review by ddeeptimahanti — APPROVED** [review]
`ddeeptimahanti@setonix-01:~> module --show_hidden av arpack

----------------------------- /software/setonix/2023.08/modules/zen3/gcc/12.2.0/dependencies -----------------------------
   .arpack-ng/3.8.0-fz75dzy (H)

`

**Pull request closed without merging**